akka-typed/src/main/scala/akka/typed/ScalaProcess.scala

```diff
@@ -0,0 +1,538 @@
+/**
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com/>
+ */
+package akka.typed
+
+import scala.concurrent.duration._
+import akka.{ actor ⇒ a }
+
+object ScalaProcess {
+  import ScalaProcess._
+
+  class RetriesExceeded extends RuntimeException
```

Comment:
That depends on whether the stack trace has a chance to point out where the failure occurred—will have to try.